### PR TITLE
Ethereals now charge with rads

### PR DESCRIPTION
**Pull request by spessbro**
## About The Pull Request
This pr makes it so ethereals get charged and possibly overcharged when exposed to radiation as well as consuming said radiation. It is to be noted that this behavior is related to their special stomachs and has been thoroughly tested with other human species.

## Why It's Good For The Game
adds some flavorful interaction between ethereals and radiation.

## Changelog
:cl:
add: Ethereal stomachs turn radiation into electric energy
/:cl:

### CODE REVIEW

**Comment by ArcaneMusic** [comment]
You need maintainer permission to make a species change. Do you have that?

**Comment by justinbooj9** [comment]
i like the concept of it, especially with the stereotype of ethereals being engineers. though depending on the charge rate it might be overpowered

File: code/modules/surgery/organs/stomach.dm [hunk 1]
```diff
@@ -144,7 +144,11 @@
 
 /obj/item/organ/stomach/ethereal/on_life()
 	..()
-	adjust_charge(-ETHEREAL_CHARGE_FACTOR)
+	if(owner.radiation)
+		owner.radiation = max(0,owner.radiation-(ETHEREAL_CHARGE_FACTOR*ETHEREAL_RAD_MULT))
```
Comment (wesoda25) [inline code/modules/surgery/organs/stomach.dm:148]: What is the point of this?

Comment (spessbro) [reply code/modules/surgery/organs/stomach.dm:148]: they don't consume charge as long as they're irradiated. It's partially to limit proc calls.

Comment (wesoda25) [reply code/modules/surgery/organs/stomach.dm:148]: I was referring to line 148. All you're doing is subtracting by 8 so I'm kinda confused as to what that will accomplish given how high rads can get.

Comment (spessbro) [reply code/modules/surgery/organs/stomach.dm:148]: oh it's to lower their rads level over time so rads aren't a death sentence

**Comment by optimumtact** [comment]
no sign of maintainer pre approval

**Pull request closed without merging**